core/src/main/java/hudson/logging/LogRecorder.java

```diff
@@ -401,6 +407,8 @@ public synchronized void doDoDelete(StaplerResponse rsp) throws IOException, Ser
      * RSS feed for log entries.
      */
     public void doRss( StaplerRequest req, StaplerResponse rsp ) throws IOException, ServletException {
+        Jenkins.get().checkPermission(Jenkins.SYSTEM_READ);
```

Comment:
Redundant given https://github.com/jenkinsci/jenkins/pull/4530/files#diff-cb8b7584e7a539c988ae99e9951f7b65R213 and https://github.com/jenkinsci/jenkins/pull/4530/files?w=1#diff-fcc5c10b76c1e25e4ac9caae4fa96acfR71 ?